### ISSUE TITLE
Consider moving all Docker images to the project's organization

### ISSUE THREAD

**Issue by jcgruenhage** [joseluisq/static-web-server#225]
**Is your feature request related to a problem? Please describe.**
Having container images tied to a person is a bit problematic for long term stability of the project I think, and considering it's already moved to an organisation on GitHub, it'd probably be a good step to do the same on docker hub, or to just use GitHub Container Registry directly instead of relying on a third party.

**Describe the solution you'd like**
Instead of pushing to `docker.io/joseluisq/static-web-server`, consider pushing to `docker.io/static-web-server/static-web-server` or `ghcr.io/static-web-server/static-web-server`.

**Describe alternatives you've considered**
None

**Additional context**
I've started using `sws` in a private context (hosting https://jcg.re on it) and considering it in a professional context as well. For that, I'd like to make sure it's not just a single person side project though, and the container image being tied to a single person indicates that to me.

EDIT: Just to make sure: I don't mean "just a single person side project" as a way to talk down on the project. It's a tremendous project and I'm very happy with it, but for professional usage we'd want it to be somewhat stable from an organizational standpoint as well. A BDFL introduces a single point of failure here, which is the reason why I'm somewhat hesitant when I see projects with one instead of a group of people maintaining it. At the same time, I of course see that finding people who are capable and willing to maintain software and that are trustworthy enough to give them commit access is also difficult.

The reason I'm looking into `sws`, is because I want an http server that handles serving static files, is written in a memory safe, highly performant language, but not much more. Caddy is what I've been using before, but with dynamic configuration, reverse proxy features, built-in ACME and so many other features, it's not quite what I want for the job here. `sws` isn't a perfect fit either, considering it handles TLS and such things as well, but it's scope is still a lot more limited than what Caddy does.

**Comment by joseluisq** [comment]
Sure, in fact, this was one of the leftovers of our last organizational move.

About publishing the images to GHCR or DockerHub organization I will have a look because it could be both for user variety of options.

I will share more here once I have the time.

**Comment by joseluisq** [comment]
As I said before, I'm interested in providing Docker images via DockerHub as well as Github Container Registry.
Here is the plan of how to proceed with this.

**DockerHub**

Unfortunately, having an organization for my current user on DockerHub is not as easy and flexible as having one on GitHub for example. And if I want that then I could incur [paying monthly](https://docs.docker.com/docker-hub/orgs/#create-an-organization) which is something that I can not afford for now.

So we will continue supporting the current `docker.io/joseluisq/static-web-server` on DockerHub.

**GitHub Container Registry**

Here is where we could provide the Docker images for the current Github organization via `ghcr.io/static-web-server/static-web-server`.
This should be enough as an alternative for organizations interested in the project as mentioned in the feature request.

--

In summary, the Docker images via DockerHub remain the same and we will support new images for the GitHub organization via ghcr.io.

So stay tuned.

**Comment by joseluisq** [comment]
This is already resolved by #232. So ghcr.io Docker images will be a thing in the next release.

**Comment by joseluisq** [comment]
Resolved on [v2.20.0](https://github.com/static-web-server/static-web-server/releases/tag/v2.20.0)
